modules/observe/proc-admin-enabling-metrics-ocp-operator.adoc

```diff
@@ -25,24 +25,26 @@ Currently, the {product} Operator does not support creating a `ServiceMonitor` c
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
-  name: <custom_resource_name> # <1>
-  namespace: <project_name> # <2>
+  name: <service-monitor-name> <1>
+  namespace: <namespace> <2>
   labels:
-    app.kubernetes.io/instance: <custom_resource_name>
+    app.kubernetes.io/instance: <service-monitor-name> <1>
     app.kubernetes.io/name: backstage
 spec:
   namespaceSelector:
     matchNames:
-      - <project_name>
+      - <namespace> <2>
   selector:
     matchLabels:
-      rhdh.redhat.com/app: backstage-<custom_resource_name>
+      app.kubernetes.io/instance: <custom_resource_name> <3>
+      app.kubernetes.io/name: backstage
   endpoints:
   - port: http-metrics
     path: '/metrics'
 ----
-<1> Replace `<custom_resource_name>` with the name of your {product} CR.
-<2> Replace `<project_name>` with the name of the {ocp-short} project where your {product} instance is running.
+<1> The name of your ServiceMonitor resource, for example, `developer-hub-service-monitor`.
+<2> The namespace where your ServiceMonitor lives and where your {product-very-short} instance is installed, for example, `rhdh-operator`.
```

Comment:
Use the `{my-product-namespace}` attribute. We just fixed all the codebase in that direction. See https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/858